README.md

```
@@ -39,97 +39,80 @@ Provide an inclusive, multilingual platform that:
-  <li><b>package-lock.json</b>, <b>package.json</b>: NPM package configuration and dependency lockfiles</li>
-  <li><b>postcss.config.js</b>: PostCSS configuration file</li>
-  <li><b>tailwind.config.js</b>: Tailwind CSS configuration</li>
-  <li><b>vercel.json</b>: Vercel deployment configuration</li>
-  <li><b>vite.config.js</b>: Vite build and tooling configuration</li>
-</ul>
-
-
-
-## Project Flowchart
-<img width="1125" height="760" alt="Untitled diagram-2025-10-11-075010" src="https://github.com/user-attachments/assets/5a406a55-d242-4b7f-9aec-de0b0194a896" />
+  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" width="100%">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)